tests/plugins/toolsets/test_tool_kafka.py

```
@@ -222,3 +227,218 @@ def test_tool_error_handling(kafka_toolset):
+
+def test_consume_messages(kafka_toolset, test_topic, admin_client):
+    """ConsumeMessages should consume messages from a topic."""
+    # Create a new topic for this test to ensure clean state
+    import uuid
+    unique_topic = f"test_consume_{uuid.uuid4().hex[:8]}"
+    
+    # Create the topic
+    from confluent_kafka.admin import NewTopic
+    new_topics = [NewTopic(unique_topic, num_partitions=1, replication_factor=1)]
+    fs = admin_client.create_topics(new_topics, validate_only=False)
+    for topic, f in fs.items():
```

Ruff (0.15.9)

[warning] Loop control variable `topic` not used within loop body

Rename unused `topic` to `_topic`

(B007)